Web.md

```diff
@@ -14,6 +14,12 @@ than if the module was JavaScript.
 More concretely, the following is a list of points of contact between WebAssembly
 and the rest of the Web platform that have been considered:
 
+## The `Wasm` object
+
+WebAssembly is exposed to the Web through a `Wasm` object, whose API is designed
```

Comment:
(Sorry, done with GDC crazy-times now.)  Yes, that makes sense.  How about "In addition to [ES6 Modules] integration, WebAssembly is exposed to JavaScript through..."  That'd lgtm